### PR TITLE
Add fling to Scrollable.

### PR DESCRIPTION
**Pull request by apwilson**

### CODE REVIEW

File: sky/packages/sky/lib/src/widgets/scrollable.dart [hunk 1]
```diff
@@ -163,6 +163,10 @@ abstract class Scrollable extends StatefulComponent {
     return scrollTo(newScrollOffset, duration: duration, curve: curve);
   }
 
+  void fling(Offset velocity) {
+    _handleDragEnd(velocity);
+  }
```
Comment (abarth) [inline sky/packages/sky/lib/src/widgets/scrollable.dart]: I'd flip this around so that _handleDragEnd calls `fling`

Comment (HansMuller) [inline sky/packages/sky/lib/src/widgets/scrollable.dart]: I agree about the flip. LGTM

Comment (apwilson) [inline sky/packages/sky/lib/src/widgets/scrollable.dart]: I almost did that but saw another class had a function called _handleDragEnd and thought maybe it was overriding it.

Comment (apwilson) [inline sky/packages/sky/lib/src/widgets/scrollable.dart]: Specifically PageableList

Comment (apwilson) [inline sky/packages/sky/lib/src/widgets/scrollable.dart]: Oh wait, I think I understand what you mean.  I'll do that

Comment (HansMuller) [inline sky/packages/sky/lib/src/widgets/scrollable.dart]: Good point. It's definitely overriding _handleDragEnd().

Comment (HansMuller) [inline sky/packages/sky/lib/src/widgets/scrollable.dart]: So PageableList would override fling() instead of _handleDragEnd().  That seems OK to me.

**Pull request merged**